docs/prover/cli/options.md

```diff
@@ -90,11 +90,14 @@ you add a new rule to an existing specification. The other is when code changes
 cause a specific rule to fail; in the process of fixing the code, updating the
 rule, and understanding counterexamples, you likely want to verify only that
 specific rule.
-
-One can either specify a specific rule name, or use pattern matching with a `*`.
-
 You can specify this flag multiple times to filter in several rules or rule patterns.
 
+**Rule Name Pattern**
+Rule name, like all CVL identifiers, has the same format as Solidity identifiers, combinations of letters, digits, 
```

Comment:
```suggestion
Rule names, like all CVL identifiers, have the same format as Solidity identifiers: they consist of a combination of letters, digits, 
```
1. Plural, just like solidity identifiers.
2. Added colon to improve sentence structure